ci-overview.md

```diff
@@ -38,6 +38,22 @@ Special commands are provided that can be included in commit messages to direct
 
 Use these commands judiciously. While they offer flexibility, they should be used appropriately to maintain the codebase's integrity and quality.
 
+### Temporarily Overriding the Pull Requeest Matrix
+
+If a workflow named `override` exists in the matrix.yaml file, this matrix will be used for pull requests instead of the `pull_request` matrix.
+This is useful for reducing resource usage when launching many CI workflows from a PR (for example, while testing CI features).
+The overridden CI job will be marked as a failure until the override is removed.
+
+Example:
+
+```
+workflows:
+  override:
```

Comment:
question: Do we think it's worth the extra complexity in the workflow implementation to add this feature? 

Typically to accomplish this functionality I'll just comment out sections of the `matrix.yaml`. Given that the `override:` requires modifying the `matrix.yaml` file anyways, I'm leaning towards thinking that this isn't worth the extra complexity. 

I'm open to having my mind changed if there's value in this functionality that I'm not seeing.

Reply:
I like having CI fail to prevent accidentally merging in the temp matrix. That's worthwhile IMO, and I don't think the implementation adds too much complexity. 

Making this an documented feature might encourage folks to use it more when it makes sense.

Reply:
> I like having CI fail to prevent accidentally merging in the temp matrix.

Nothing stops me from just ignoring `override:` and just modifying the matrix anyways, right?